Gemfile.lock

```diff
@@ -318,6 +318,7 @@ DEPENDENCIES
   license_finder (~> 5.0.0)
   listen (~> 3.1.5)
   mini_magick (~> 4.8.0)
+  mocha (~> 1.3.0)
```

Comment:
@veronicacannon Shouldn't the mocha gem be added to the Gemfile in addition to the Gemfile.lock?

Reply:
@BrendaPenn yes! The workflow is: 1) add file to Gemfile, 2) run `bundle install`(this step modifies the Gemfile.lock)

Reply:
(sorry for jumping in 😸 )

Reply:
Sorry I missed this @BrendaPenn Thanks @elskwid

Reply:
Rebased to latest Gemfile and ready for approval.